src/shared/lib/oql/oql_basic.pegjs

```diff
@@ -28,17 +30,19 @@ MUT = "MUT"i
 EXP = "EXP"i
 PROT = "PROT"i
 
+SuperQuery
```

Comment:
i dont like this name :) whats next, SuperSuperQuery? I say call this Query and rename whats now Query to something else

Reply:
Good point. I changed it to `Query` and the old `Query` is now called `StandardQuery`. See commit.